### DISCUSSION TITLE
Current marker with active class for dropdown-toggle Title

### DISCUSSION THREAD

**Discussion by Allmedialab** [twbs/bootstrap#30841] [General]
Hi,

I can't find this in the documentation, but how do I show the active class on the dropdown Title when clicking on a Sub item. So the result is that `nav-link dropdown-toggle` is showing the current marker for both the Title as the clicked `dropdown-item` (Submenu). Nothing seems to work.

Regards & Thanks,

David 
 
```html
<li class="nav-item dropdown">
   <a class="nav-link dropdown-toggle" href="#" id="dropdown02" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Title</a>
  <div class="dropdown-menu" aria-labelledby="dropdown01">
    <a class="dropdown-item" href="../nl/arrangementen">Submenu 1</a>
    <a class="dropdown-item" href="../nl/koffietafel">Submenu 2</a>
    <a class="dropdown-item" href="../nl/kinderfeestjes">Submenu 3</a>
  </div>
</li>
```

**Comment by mdo** [comment]
You'd need to use some custom JavaScript for that—we don't provide that option out of the box in Bootstrap. If you want that kind of functionality, you could use the `.form-select` or `.custom-select` and apply additional styles if you need it to look like a button.
